adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.h

```diff
@@ -39,5 +39,6 @@ extern const CGSize kVNGBannerShortSize;
 - (void)completeBannerAdViewForPlacementID:(nonnull id<GADMAdapterVungleDelegate>)delegate;
 - (BOOL)isSDKInitialized;
 - (nullable NSString *)getSuperToken;
+- (void)setCOPPAStatus:(nonnull NSNumber *)coppa;
```

Comment:
Shouldn't this take in boolean instead of number?

See: https://developers.google.com/admob/ios/api/reference/Classes/GADRequestConfiguration?hl=en#-tagforchilddirectedtreatment: